
message StringMessage {
  optional string b = 2;
}